weave-js/src/react.tsx

```diff
@@ -544,7 +544,7 @@ const RE_WEAVE_CALL_REF_PATHNAME = new RegExp(
     '/',
     PATTERN_PROJECT,
     '/call/',
-    '([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12})', // Call UUID
+    '([0-9a-fA-F-]+)', // Call ID
```

Comment:
I would add a comment here with your comment from the PR about this being for OTEL support

Reply:
Done